interface_tester/plugin.py

```diff
@@ -88,6 +93,10 @@ def configure(
         :param config: charm config.yaml contents.
         :param juju_version: juju version that Scenario will simulate (also sets JUJU_VERSION
             envvar at charm runtime.)
+        :param interface_subdir: Subdirectory to look for versioned interface direstories in
+            under interfaces/interface_name.
+        :param tests_dir: Name of tests directory under
+            interfaces/interface_name/interface_subdir/vN.
```

Comment:
perhaps it should be more clear what's a literal and what's a variable?
```suggestion
            interfaces/<interface_name>/<interface_subdir>/v<N>.
```